packages/p2p/src/components/my-profile/payment-methods/add-payment-method/add-payment-method.scss

```diff
@@ -16,6 +16,14 @@
         display: none;
     }
 
+    &-select {
+        height: 30vh;
```

Comment:
Are you using this for desktop?

Reply:
Yeah, i had to add it for the select payment method dropdown, since there was an issue with it, for both mobile and desktop there seems to be no issue

<img width="449" alt="Screenshot 2022-08-04 at 4 32 16 PM" src="https://user-images.githubusercontent.com/103412909/182801942-f1f38397-e532-45fe-8a8c-a8b840f65c84.png">